walletunlocker/service_test.go

```diff
@@ -365,3 +365,103 @@ func TestUnlockWallet(t *testing.T) {
 		t.Fatalf("password not received")
 	}
 }
+
+// TestChangeWalletPassword tests that we can successfully change the wallet's
+// password needed to unlock it.
+func TestChangeWalletPassword(t *testing.T) {
+	t.Parallel()
+
+	// testDir is empty, meaning wallet was not created from before.
+	testDir, err := ioutil.TempDir("", "testchangepassword")
+	if err != nil {
+		t.Fatalf("unable to create temp directory: %v", err)
+	}
+	defer func() {
+		os.RemoveAll(testDir)
```

Comment:
nit: can be made `defer os.RemoveAll(testDir)`

Reply:
Fixed.